portal.occ-data.org/manifest.json

```diff
@@ -8,22 +8,13 @@
   },
   "versions": {
     "arborist": "quay.io/cdis/arborist:2021.06",
-    "aws-es-proxy": "quay.io/cdis/aws-es-proxy:0.8",
     "fence": "quay.io/cdis/fence:2021.06",
     "indexd": "quay.io/cdis/indexd:2021.06",
     "peregrine": "quay.io/cdis/peregrine:2021.06",
-    "pidgin": "quay.io/cdis/pidgin:2021.06",
```

Comment:
the integration tests are failing because this service is missing 🙁

Reply:
maybe we can add a check not to run `@coreMetadata` when `pidgin` is not running - or add a `@requires-pidgin` tag
@uc-cdis/planx-qa

Reply:
Hey @paulineribeyre thanks for letting us know. I added pidgin service and committed the changes. Can I please get a  review and merge. Thanks 😊